### PR TITLE
Add favicons-webpack-plugin@3.0.0 sub-dependency

### PR DESCRIPTION
**Pull request by mt-micky**
- [ ] Are you the package author?
- [x] Downloaded, extracted, and reviewed the contents of the package tarball on the npm public registry.
- [x] Checked `README`, `LICENSE`, `COPYING`, and header comments for conflicting or additional licenses.
- [x] Searched for "license", such as with `fgrep -ir license $PACKAGE_DIRECTORY`.
- [x] Compared the license text found to standard language on <htts://spdx.org/licenses/>.
- [ ] Asked the package author about the applicable license.

Newest version of those packages are explicitly set as MIT in their `package.json` file

### CODE REVIEW

File: index.csv [hunk 1]
```diff
@@ -225,3 +225,6 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+rework,MIT,1.0.1
+process,MIT,0.5.2
```
Comment (ljharb) [inline index.csv]: this also applies to v0.6.0, v0.7.0, v0.8.0, v0.9.0, v0.10.0, v0.10.1, v0.11.0, and v0.11.1 - can you add those entries as well?

File: index.csv [hunk 2]
```diff
@@ -225,3 +225,6 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+rework,MIT,1.0.1
```
Comment (ljharb) [inline index.csv]: no version of rework contains a license declaration or a license file that i can see; is there a link to somewhere the author has explicitly stated that a license should retroactively apply?

File: index.csv [hunk 3]
```diff
@@ -225,3 +225,6 @@ longest,MIT,1.0.1
...
+process,MIT,0.5.2
+exif-parser,MIT,0.1.12
```
Comment (ljharb) [inline index.csv]: this also applies to v0.1.11, v0.1.10, v0.1.9, v0.1.8, v0.1.7, v0.1.6, v0.1.5, v0.1.4, v0.1.3, v0.1.2, v0.1.1, and v0.1.0 - can you add those entries as well?

**Review by ljharb — CHANGES_REQUESTED** [review]

File: index.csv (hunk 2 above)
Comment (mt-micky) [reply index.csv]: It's in the Readme, isn't it?

Comment (mt-micky) [reply index.csv]: If we don't allow Readme file entry for license let me know, I will remove `rework` from here

Comment (ljharb) [reply index.csv]: aha, thanks - https://github.com/reworkcss/rework#license (also available in the npm tarballs).

In that case, let's also add entries for v1.0.0, v0.20.3, v0.20.2, v0.20.1, v0.20.0, v0.19.0, v0.18.3, v0.18.2, v0.18.1, v0.18.0, v0.17.4, v0.17.3, v0.17.2, v0.17.1, v0.17.0, v0.16.0, v0.15.0, v0.14.0, v0.13.2, v0.13.1, v0.13.0, v0.12.2, v0.12.1, v0.12.0, v0.11.0, v0.10.2, 0.10.1, v0.10.0, v0.9.2, v0.9.1, v0.9.0, v0.8.0, v0.7.0, v0.6.1, v0.6.0, v0.5.1, v0.5.0, v0.4.1, v0.4.0, v0.3.0, v0.1.2, v0.1.1, v0.1.0, and v0.0.1.

**Review by ljharb — APPROVED** [review]

**Pull request merged**